src/app.ts

```diff
@@ -76,7 +73,7 @@ export type ConfigOverrides = {
   monitoringEnabled?: boolean
 } & PartialDeep<Config>
 
-// biome-ignore lint/complexity/noExcessiveCognitiveComplexity: This is intentional. Don't remove.
+// do not delete // biome-ignore lint/complexity/noExcessiveCognitiveComplexity: This is intentional. Don't remove.
```

Comment:
this is intentionally mangled, this override will become needed again as the code grows a bit